frontend/src/pages/RunDetails.test.tsx

```diff
@@ -1091,9 +1091,10 @@ describe('RunDetails', () => {
           >
             <Banner
               additionalInfo="getting logs failed"
-              message="Warning: failed to retrieve pod logs. Possible reasons include cluster autoscaling or pod preemption"
+              message="Failed to retrieve pod logs. Possible reasons include cluster autoscaling or pod preemption. Use Stackdriver Kubernetes Monitoring to view them."
```

Comment:
Does it show the exclamation mark when mode is warning? Can we change the mode to info and rephrase the message? Something like "Failed to retrieve pod logs. Possible reasons include cluster autoscaling, pod garbage collection, and pod preemption. Use Stackdriver Kubernetes Monitoring to view them."

Reply:
@jiezhang yes it does show a ⚠️ . Perhaps an info type banner does makes sense. I'll add one tomorrow

Reply:
Actually on second thoughts maybe a warning makes sense since it could either be an error fetching the logs, or an info if it's just that a node as scaled down. Would it make sense to dig down deeper to know what has actually happened?

Reply:
Totally agree we should dig deeper on the error to know if it's expected. I'm not sure how that can be done, @jonasdebeukelaer would you want to investigate?

I think if we cannot know why, we need to keep the warning sign. The new sentence of "Use Stackdriver Kubernetes Monitoring to view them" is already a big step forward indicating users another option.

Maybe we should make the stackdriver link stand out when pod log cannot be achieved, what do you think?

Reply:
sure, happy to have a look. 

So logic is:
* show logs if can
* if not
  * if node skipped - hide logs tab
   * if run is running/finished and pod can't be found - info banner saying that logs have gone w possible reasons, and optional 'access via stackdriver'
   * else error banner

and yes I can give the stackdriver link option more presence

Reply:
Thanks! This sounds perfect.

nit: `if run is running/finished and pod can't be found` -> `if run is in other states`, there are also like failed state that we should show log too

Reply:
Hmm, rethinking over it, I'm not sure we can get to the `else error banner` branch.

It is more of a problem that, when we get an error when fetching logs from pod, we need to parse the error message/status code and know some errors are expected (pod went away) and only show warning/error for others.

Reply:
NOTE that this happens for every pod - within hours of a pod's normal execution, it will be garbage collected. Why do we want to display a warning for 100% of the steps?

Reply:
@jiezhang I've converted it to an info type banner, if a get to 'api/pod' returns a 404, so no more warnings. it will show an error if the logs don't display for a different reason though.